.gitlab-ci.yml

```diff
@@ -3,7 +3,6 @@ include:
 
 Unit Test:
   image: quay.io/ebi-ait/ingest-base-images:trion_ng-cli-karma_11.2.6
```

Comment:
No stage name here?

Reply:
Actually not necessary, I can remove it from the other. The jobs merge so stage name will be merged from the template